src/main/kotlin/org/elasticsearch/client/KotlinExtensions.kt

```diff
@@ -127,7 +128,8 @@ inline fun <reified T : Any> RestHighLevelClient.indexRepository(
     readAlias: String = index,
     writeAlias: String = index,
     refreshAllowed: Boolean = false,
-    defaultRequestOptions: RequestOptions = RequestOptions.DEFAULT
+    defaultRequestOptions: RequestOptions = RequestOptions.DEFAULT,
```

Comment:
Lets keep the request options last.

Reply:
Aha, at first I thought you pushed a commit addressing this before  merging this into master. But now since this is already merged I'm not sure, shall I create another MR that puts the request options last?

Reply:
Here's another PR that moves the RequestOptions to the last place.
https://github.com/jillesvangurp/es-kotlin-wrapper-client/pull/25